source/clear-linux/guides/maintenance/assign-static-ip.rst

```diff
@@ -0,0 +1,71 @@
+.. _assign-static-ip:
+
+Assign a static IP address to a network interface
+#################################################
+
+Introduction
+************
+
+By default, your |CL-ATTR| system automatically gets an IP address from your
+network via DHCP. If you do not have a DHCP server on your network, assign a
```

Comment:
Pls change "If you do not have a DHCP server on your network" to 'If you do not have a DHCP server on your network or simply want to use static IP address"